pkgs/applications/gis/saga/default.nix

```diff
@@ -31,32 +31,29 @@
 
 mkDerivation rec {
   pname = "saga";
-  version = "7.9.1";
+  version = "8.4.0";
 
   src = fetchurl {
     url = "mirror://sourceforge/saga-gis/SAGA%20-%20${lib.versions.major version}/SAGA%20-%20${version}/saga-${version}.tar.gz";
-    sha256 = "sha256-Jq1LhBSeJuq9SlNl/ko5I8+jnjZnLMfGYNNUnzVWo7w=";
+    sha256 = "sha256-v6DPwV20fcsznrEaFJk0/ewU4z3cTjzYYuLkyMwSLV0=";
   };
 
+  sourceRoot = "saga-${version}/saga-gis";
+
   nativeBuildInputs = [
     # Upstream's gnerated ./configure is not reliable
```

Comment:
```suggestion
```
This comment doesn't make sense if we're building with cmake.

Reply:
Removed.